content/enterprise_influxdb/v1.9/administration/backup-and-restore.md

```diff
@@ -316,12 +316,12 @@ Restored from my-incremental-backup/ in 83.892591ms, transferred 588800 bytes
 
 ##### Restore from a metadata backup
 
-In this example, the `restore` command restores an metadata backup stored
+In this example, the `restore` command restores a metadata backup stored
 in the `metadata-backup/` directory.
 
 ```bash
 # Syntax
-influxd-ctl restore <path-to-backup-directory>
+influxd-ctl restore -meta-only-overwrite-force <path-to-backup-directory>
```

Comment:
This is wrong - see above, we want more of a note to distinguish between the two cases:

1. EXISTING DOCS: Generate a metadata-only backup, and do a full restore with it. This re-numbers all the shards, but is safe to do as it can only be done in an empty cluster. Use case is moving existing users, CQ's, other metadata to a new cluster

2. NEW DOCS: Do a -meta-only-overwrite-force restore from any kind of backup. This overwrites the full metadata state, and should not be done across clusters, only with a backup from an existing cluster. It is for the very specific circumstance where there are bad meta nodes but the data nodes in the cluster are still good, and we want to overwrite the meta state to some recent state without touching the data nodes. Note: if you are not restoring into the same cluster (with same meta node hostnames etc) that the backup is from, bad things could happen.